packages/esm-patient-chart-app/src/ui-components/forms-side-rail.component.tsx

```diff
@@ -0,0 +1,46 @@
+import React, { useState } from 'react';
+import { Button } from 'carbon-components-react';
+import Document20 from '@carbon/icons-react/es/document/20';
+import styles from './forms-side-rail.scss';
+import { ScreenModeTypes, WindowSize } from '@openmrs/esm-patient-common-lib';
+import Close20 from '@carbon/icons-react/es/close/20';
+import WarningFilled16 from '@carbon/icons-react/es/warning--filled/16';
+import { patientChartWorkspaceSlot } from '../constants';
+import { useAssignedExtensionIds } from '@openmrs/esm-framework';
+
+interface FormsSideRailProps {
+  checkViewMode: (active: boolean) => void;
+  windowSize: WindowSize;
+}
+
+const FormsSideSideRail: React.FC<FormsSideRailProps> = ({ checkViewMode, windowSize }) => {
```

Comment:
I'm confused. This thing is called "side rail" but it looks like it's just a button?